packages/odyssey-react-mui/src/Icon.tsx

```diff
@@ -27,14 +31,26 @@ export interface IconProps extends SvgIconNoChildrenProps {
  * This component includes all icon data as static imports. Favor individual
  * icon component imports where possible to keep your bundle size smaller.
  */
-export const Icon = forwardRef<SVGSVGElement, IconProps>(
-  ({ name, titleAccess, ...rest }, ref) => {
-    if (!(name in iconDictionary)) return null;
+export const Icon = ({
+  name,
+  size,
+  className,
+  titleAccess,
+  ...rest
+}: IconProps) => {
+  if (!(name in iconDictionary)) return null;
 
-    const NamedIcon = iconDictionary[name];
+  const NamedIcon = iconDictionary[name];
 
-    return <NamedIcon titleAccess={titleAccess} ref={ref} {...rest} />;
-  }
-);
+  return (
+    <NamedIcon
+      className={`icon Icon-size${
+        size.charAt(0).toUpperCase() + size.slice(1)
+      }`}
+      titleAccess={titleAccess}
+      {...rest}
+    />
```

Comment:
I'd like to avoid using `className` here if possible. I know that with MUI theming, this might be more difficult.

Reply:
What about something like

```
size: IconProps["fontSize"]

...

<NamedIcon fontSize={size}>
```

Then in `components.tsx` we can set the sizes like:

```
...(ownerState.fontSize === "medium" && {
  fontSize: theme.whatever,
}),
```